### DISCUSSION TITLE
Welcome to foam Discussions!

### DISCUSSION THREAD

**Discussion by riccardoferretti** [foambubble/foam#948] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lloyd-Jackman-UKPL** [comment]
Hey

I'm Lloyd, an Englishman in Gdynia, Poland working as a product manager in the financial services industry when not performing the infinitely greater challenge of being a good Dad. I've self taught myself to code in Python, but only as a means of doing more interesting (and valuable) work with my time and have no ambitions to become a programmer, though I have had the pleasure of working with many talented programmers.

I'm using Foam to organise my notes at work primarily. I work in a very complex and fast moving environment and without good notes, I'd quickly get lost or lost what matters. The ease with which I can link ideas and view those relationships is very important to me for solving problems and ideation. The flexibility allows me to create and recreate my view of the world, without getting boxed into false hierarchies. Finally, the control and privacy of keeping things in local markdown files is not only good practice but a necessity for the industry, with my notes staying mine.

Foam is an excellent project and whilst it lacks some of the polish of Obsidian or Roam, it has plenty of potential as a part of a rich VS ecosystem.
